sdk/core/azure-core/inc/azure/core/uuid.hpp

```diff
@@ -42,7 +50,7 @@ namespace Azure { namespace Core {
      * representation of the Uuid.
      * @returns An array with the binary representation of the Uuid.
      */
-    std::array<uint8_t, UuidSize> const& AsArray() const { return m_uuid; }
+    constexpr ValueArray const& AsArray() const { return m_uuid; }
```

Comment:
Just to confirm, changing to an alias like this is never a breaking change?

Reply:
No, it is not.